src/app/workflow/workflow.component.html

```diff
@@ -106,7 +106,11 @@ <h3>
       <span *ngIf="(categories$ | async).length > 0" class="inline-block mr-5 mt-1 mb-1">
         <span class="size-small mr-2">Categories</span>
         <mat-chip-list class="bubble-list">
-          <app-category-button *ngFor="let category of (categories$ | async)" [category]="category" entryType="workflow"></app-category-button>
+          <app-category-button
```

Comment:
For the sake of knowledge, was there anything wrong with this tag, besides the extra parens?

Reply:
I doubt it, this is one of the automatically husky reformatings